src/data/queries/healthCareLocalFacility.ts

```diff
@@ -134,5 +138,13 @@ export const formatter: QueryFormatter<
       fieldInstagram: entity.field_region_page.field_instagram ?? null,
       fieldYoutube: entity.field_region_page.field_youtube ?? null,
     },
+    lovellVariant: lovell?.variant,
+    lovellSwitchPath:
+      entity.field_main_location && lovell?.variant
```

Comment:
This is the part that I couldn't figure out how content-build was doing it. I got bored digging around over there and solved it using this value from Drupal. It seemed like an explicit way to set the switch path...or not set it while preserving all the rest of the Lovell properties.